scripts/minhook.cmake

```diff
@@ -2,8 +2,8 @@ include(FetchContent)
 
 FetchContent_Declare(
     minhook
-    GIT_REPOSITORY https://github.com/YimMenu/minhook.git
-    GIT_TAG        902ab63e0a771547961e132ccc0700d62e2a1423
+    GIT_REPOSITORY https://github.com/TsudaKageyu/minhook.git
```

Comment:
Please use the yim branch for custom scripts not someone else's or your own branch